nss-day-cohort-51/nutshell-sugar-deadies
# Feature/nutshell 10

# Description

## Type of change


# Testing Instructions
Click events in navbar, check if events show in chronological order, with the to event having larger font and bolded. click create new event and save and see that new event it saved. Delete an event. and click show weather to see weather.

# Checklist:
